### PR TITLE
[zh-cn] Update CSS fragmentation introduction

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

revise translation of 'inline' and update the document

### Related issues and pull requests

Part of #12632

### CODE REVIEW

File: files/zh-cn/web/css/css_fragmentation/index.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
 ---
-title: CSS 分片
+title: CSS 片段
 slug: Web/CSS/CSS_fragmentation
 ---
 
 {{CSSRef}}
 
-**CSS Fragmentation** 是 **CSS** 的模块，它定义了内容在跨多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)，区域或[列中](/zh-CN/docs/Web/CSS/CSS_multicol_layout)被分割（分段）时如何显示。
+**CSS 片段**模块定义了当内容在多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)、区域或[列](/zh-CN/docs/Web/CSS/CSS_multicol_layout)中被分割（fragmented）时的显示方式。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/css_fragmentation/index.md:8]: Fragmentation Module 翻译参考了w3c的译法

https://www.chinaw3c.org/archives/2255/

File: files/zh-cn/web/css/css_fragmentation/index.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 
-当一个内联框包装成多行时会发生碎片。当一个块跨越一个列布局容器内的多个列，或者在打印时跨越一个分页符时，也会发生这种情况。元素的每个渲染*片段*称为一个*片段*。
+当行内盒子被分割到多行上时，就会出现分片。当一个块在列布局容器内跨越多列，或在打印时跨越分页符时，也会出现这种情况。元素的每个呈现片段都称为*分片*。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/css_fragmentation/index.md:10]:
```suggestion
当行向盒子被分割到多行上时，就会出现分片。当一个块在列布局容器内跨越多列，或在打印时跨越分页符时，也会出现这种情况。元素的每个呈现片段都称为*分片*。
```

**Comment by jasonren0403** [comment]
'lint-and-review-docs' isn't running well 🤔 I'm not sure why... @yin1999 
image

File: files/zh-cn/web/css/css_fragmentation/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/css/css_fragmentation/index.md:8]:
```suggestion
**CSS 片段**模块定义了当内容被被分割（fragmented）到多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)、区域或[列](/zh-CN/docs/Web/CSS/CSS_multicol_layout)时的显示方式。
```

File: files/zh-cn/web/css/css_fragmentation/index.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
 slug: Web/CSS/CSS_fragmentation
 ---
 
 {{CSSRef}}
 
-**CSS Fragmentation** 是 **CSS** 的模块，它定义了内容在跨多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)，区域或[列中](/zh-CN/docs/Web/CSS/CSS_multicol_layout)被分割（分段）时如何显示。
+**CSS 片段**模块定义了当内容在多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)、区域或[列](/zh-CN/docs/Web/CSS/CSS_multicol_layout)中被分割（fragmented）时的显示方式。
 
-当一个内联框包装成多行时会发生碎片。当一个块跨越一个列布局容器内的多个列，或者在打印时跨越一个分页符时，也会发生这种情况。元素的每个渲染*片段*称为一个*片段*。
+当行向盒子被分割到多行上时，就会出现分片。当一个块在列布局容器内跨越多列，或在打印时跨越分页符时，也会出现这种情况。元素的每个呈现片段都称为*分片*。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_fragmentation/index.md:10]: The translation of "fragmentation" is not align with the context.

**Review by yin1999 — COMMENTED** [review]
Some comments here

File: files/zh-cn/web/css/css_fragmentation/index.md (hunk 3 above)
Comment (jasonren0403) [reply files/zh-cn/web/css/css_fragmentation/index.md:10]:
```suggestion
当行向盒子被分割到多行上时，就会出现片段。当一个块在列布局容器内跨越多列，或在打印时跨越分页符时，也会出现这种情况。元素的每次呈现都称为*片段*。
```

File: files/zh-cn/web/css/css_fragmentation/index.md [hunk 4]
```diff
@@ -1,16 +1,18 @@
 ---
-title: CSS 分片
+title: CSS 片段
 slug: Web/CSS/CSS_fragmentation
 ---
 
 {{CSSRef}}
 
-**CSS Fragmentation** 是 **CSS** 的模块，它定义了内容在跨多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)，区域或[列中](/zh-CN/docs/Web/CSS/CSS_multicol_layout)被分割（分段）时如何显示。
+**CSS 片段**模块定义了当内容被被分割（fragmented）到多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)、区域或[列](/zh-CN/docs/Web/CSS/CSS_multicol_layout)时的显示方式。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_fragmentation/index.md:8]:
```suggestion
**CSS 片段**模块定义了当内容被分割（fragmented）到多个[页面](/zh-CN/docs/Web/CSS/CSS_paged_media)、区域或[列](/zh-CN/docs/Web/CSS/CSS_multicol_layout)时的显示方式。
```

**Review by yin1999 — APPROVED** [review]
Sorry, I didn't explain clearly. Please consider this. If it makes sense to you, just go ahead and merge this.

Thanks @jasonren0403

**Pull request merged**